test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -97,7 +97,8 @@ Describe "Stream writer tests" -Tags "CI" {
            }
            else
            {
-               $result.Computer | Should Be $(uname -n)
+               # Use Match instead of Be so we can avoid dealing with a potential domain name
+               $result.Computer | Should Match ".*$(uname -n)"
```

Comment:
Why not use `hostname` instead?

Also, wouldn't a FQDN regex be "$(hostname)\.?.*"?  so that foo matches foo.microsoft.com, but doesn't match foobar.microsoft.com?